test/actual-uploads.test.js

```diff
@@ -0,0 +1,393 @@
+import { sha256 } from '@noble/hashes/sha256'
+import fetch from '@web-std/fetch'
+import http from 'http'
+import { assert, beforeEach, describe, expect, it, test } from 'vitest'
+
+import { SigV4 } from '../src/index.js'
+import { badFetch, badFetchSocket } from './badFetch.js'
+import { encodeBase64, sleep } from './utils.js'
+
+require('dotenv').config()
+
+describe('Signer', function () {
+  describe('s3 integration needs .env and cors setup', function () {
```

Comment:
Is this to warn devs that they need to do some manual set up to run these tests? If so, please could this be documented in the README instead?